### DISCUSSION TITLE
Can format like Wikipedia people pages?

### DISCUSSION THREAD

**Discussion by JiveyGuy** [requarks/wiki#5845] [Help / Questions] [answered]
Hey there, trying to get a right aligned table with images and information like they have for people pages on wikipedia, here is an example: 
![Screenshot from 2022-11-15 19-05-45](https://user-images.githubusercontent.com/20270893/202052137-2b7845b3-1788-411b-8b65-5065f037a8c3.png)

Is this possible with wiki.js?

**Comment by JiveyGuy** [comment]
No progress getting this format on my end, any thoughts?

**Reply by NGPixel** [reply, marked as answer]
No. This will be added in a future update.
